repository_service_tuf_worker/repository.py

```diff
@@ -823,22 +833,27 @@ def bump_snapshot(self) -> bool:
 
         return True
 
-    def bump_online_roles(self) -> bool:
-        """Bump online Roles (Snapshot, Timestamp, BINS)."""
-        timeout = int(self.refresh_settings().get("LOCK_TIMEOUT", 60.0))
-        logging.debug(f"Configured timeout: {timeout}")
+    def bump_online_roles(self, force: Optional[bool] = False) -> bool:
+        """
+        Bump online roles (Snapshot, Timestamp, Targets and BINS).
+
+        Args:
+            force: force target roles bump if they don't match the hours before
+                expire (`self._hours_before_expire`)
+        """
+        logging.debug(f"Configured timeout: {self._timeout}")
         status_lock_timestamp = False
         # Lock to avoid race conditions. See `LOCK_TIMEOUT` in the Worker guide
         # documentation.
         try:
-            with self._redis.lock(LOCK_TIMESTAMP, timeout=timeout):
+            with self._redis.lock(LOCK_TARGETS, timeout=self._timeout):
```

Comment:
I moved the lock key as discussed with @MVrachev.

We also need a specific issue to improve our locks and make them more reliable.
I will file an specific issue for it

Reply:
https://github.com/repository-service-tuf/repository-service-tuf-worker/issues/276